docs/02-app/01-building-your-application/07-configuring/03-environment-variables.mdx

```diff
@@ -163,13 +163,11 @@ export default function Component() {
 
 ## Default Environment Variables
 
-In general only one `.env.local` file is needed. However, sometimes you might want to add some defaults for the `development` (`next dev`) or `production` (`next start`) environment.
+Typically, only `.env*` file is needed. However, sometimes you might want to add some defaults for the `development` (`next dev`) or `production` (`next start`) environment.
 
 Next.js allows you to set defaults in `.env` (all environments), `.env.development` (development environment), and `.env.production` (production environment).
 
-`.env.local` always overrides the defaults set.
-
-> **Good to know**: `.env`, `.env.development`, and `.env.production` files should be included in your repository as they define defaults. **`.env*.local` should be added to `.gitignore`**, as those files are intended to be ignored. `.env.local` is where secrets can be stored.
+> **Good to know**: `.env`, `.env.development`, and `.env.production` files should be included in your repository as they define defaults. All `.env` files are excluded in `.gitignore` by default, allowing you to opt-into committing these values to your repository.
```

Comment:
Should we not mention `.env` here? Aren't we trying to say that `.env` is the new recommendation for secrets and such?

> All `.env` files are excluded in `.gitignore` by default, allowing you to opt-into committing these values to your repository.

This is only true if you are using the version of `create-next-app` from the version up when this PR gets merged/released. If you are on a version below, this might be confusing as your .env _will_ be committed by default still.